File: exercicios/codigosEmOutrasLinguagens/EX10REC/EX10RECv1.py
def main():
    n = int(input("Quantos números você deseja informar? "))

    numeros = []

    print("Digite os números:")
    for _ in range(n):
        numero = int(input())
        numeros.append(numero)

    print("\nOrdem inversa:")
    for i in range(n - 1, -1, -1):
        print(numeros[i])

if __name__ == "__main__":
    main()

File: exercicios/codigosEmOutrasLinguagens/EX10REC/EX10RECv1.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int n, i;

    printf("Quantos números você deseja informar? ");
    scanf("%d", &n);

    int *numeros = malloc(n * sizeof(int));
    if (numeros == NULL) {
        printf("Erro ao alocar memória.\n");
        return 1;
    }

    printf("Digite os números:\n");
    for (i = 0; i < n; i++) {
        scanf("%d", &numeros[i]);
    }

    printf("\nOrdem inversa:\n");
    for (i = n - 1; i >= 0; i--) {
        printf("%d\n", numeros[i]);
    }

    free(numeros);
    return 0;
}